modules/rhizo_base/manifests/fixes.pp

```diff
@@ -49,6 +49,7 @@
 
   file { '/var/lib/puppet/state':
       ensure  => link,
-      target  => '/var/cache/puppet/state/'
+      target  => '/var/cache/puppet/state/',
+      force   => true,
```

Comment:
Did you actually have a problem with this? The old directory really should NOT exist. If it were to exist already for some reason, I'm not sure I want to blindly replace it with a symlink.

Reply:
I did, but it was on an install based on a vagrant box, not an install based on the debian installer directly. The directory for me existed, but was empty. This could be an artifact from vagrant, in which case this definitely doesn't need to happen and I can fix at vagrant startup. Looking at this commit with a few day's hindsight I agree that force is probably not the right thing since it could blow away existing config.